java/src/org/openqa/selenium/grid/node/config/NodeFlags.java

```diff
@@ -261,6 +262,13 @@ public class NodeFlags implements HasRoles {
   @ConfigValue(section = NODE_SECTION, name = "enable-managed-downloads", example = "false")
   public Boolean managedDownloadsEnabled;
 
+  @Parameter(
+      names = {"--sub-path"},
+      arity = 1,
+      description = "A sub-path that should be considered for all user facing routes on the Node")
+  @ConfigValue(section = NETWORK_SECTION, name = "sub-path", example = "my_company/selenium_grid")
+  public String subPath;
```

Comment:
@vietnd96 - Can you please let me know what would the application behaviour be in the below scenario? 

1. Hub was started with the sub path as "/foo"
2. Node1 is now being started with the sub-path as "/bar"
3. Node2 is now being started with the sub-path as "foobar"

So does this mean that a testcase would need to know the sub-path using which the node (to which the session was routed to) was spun off, in order for it to be able to access the web sockets?